src/main/scala/cromwell/binding/WdlNamespace.scala

```diff
@@ -115,79 +87,68 @@ object WdlBinding {
   }
 
   /* All MemberAccess ASTs that are not contained in other MemberAccess ASTs */
-  def findTopLevelMemberAccesses(expr: AstNode): Iterable[Ast] = WdlBinding.findAstsWithTrail(expr, "MemberAccess").filter {
+  def findTopLevelMemberAccesses(expr: AstNode): Iterable[Ast] = AstTools.findAstsWithTrail(expr, "MemberAccess").filter {
     case(k, v) => !v.exists{case a:Ast => a.getName == "MemberAccess"}
   }.keys
 
-  def terminalMap(ast: Ast, source: WdlSource) = (findTerminals(ast) map {(_, source)}).toMap
-
-  def getCallInput(ast: Ast): Map[String, WdlExpression] = getCallInputAsts(ast).map(processCallInput).toMap
-
   def getCallInputAsts(ast: Ast): Seq[Ast] = {
     findAsts(ast, AstNodeName.Inputs) match {
-      case x: Seq[Ast] if x.size == 1 => WdlBinding.findAsts(x.head.getAttribute("map"), AstNodeName.IOMapping)
+      case x: Seq[Ast] if x.size == 1 => AstTools.findAsts(x.head.getAttribute("map"), AstNodeName.IOMapping)
```

Comment:
it matches a list of a single element, although now that I write this I realize it might only work on List and not Vector